pipeline/runtime/iacm_nudge.go

```diff
@@ -0,0 +1,140 @@
+package runtime
+
+import (
+	"errors"
+	"github.com/harness/lite-engine/logstream"
+)
+
+func getIACMNudges() []logstream.Nudge {
+	return []logstream.Nudge{
+		logstream.NewNudge("[ERROR] approval step has been rejected",
+			"Check why the approval step was rejected",
+			errors.New("the approval step has been rejected")),
+		logstream.NewNudge("[ERROR] Error parsing the init_option block",
+			"Contact Harness Engineering",
+			errors.New("there was an error starting the plugin")),
```

Comment:
can we provide a description on what caused the error?

Reply:
This specific error is one of those errors that should not happen nor the customer can do anything to alliviate it. For example this one is that when we start the plugin, one of the objects that we populate from the manager was populated wrongly. This is a manager problem literally that is been propagated to the plugin. And because is a manager problem the customers can't do a lot as this is something that needs to be probably fixed by us changing the code of the manager. This is the kind of errors that we would never expect to see in prod _but_ I think that we need to add just in case it happens and we need to investigate what is going on.